networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/libp2p/LibP2PNetwork.java

```diff
@@ -171,6 +175,7 @@ public String getNodeAddress() {
 
   @Override
   public SafeFuture<?> connect(final String peer) {
+    logger.info("Connecting to " + peer);
```

Comment:
The `PeerManager` already has logging around connections.

Reply:
Thanks